src/components/Price.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react'
+import Price from './Price'
+
+test('renders price component with data received in props', () => {
```

Comment:
It's also good habit to wrap all your tests (in a file) by a `describe`.

In this case, you could do

```js
describe('Price Component', function () {...})
```

```js
describe('<Price />', function () {...})
```

whatever you feel like is good.

Reply:
The test has been wrapped in a test suite with `describe`